deployment/ansible/roles/cac-tripplanner.app/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install packages
+  apt: name={{ item }} state=present
+  with_items:
+    - postgresql-9.3
+    - postgresql-9.3-postgis-2.1
+    - libpq-dev
+    - python-dev
+    - python-psycopg2
```

Comment:
[azavea.postgresql-support](https://github.com/azavea/ansible-postgresql-support) handles installing dependencies for the Ansible PostgreSQL modules.